GameWindow.java

```diff
@@ -26,6 +25,7 @@
 import javax.swing.JRadioButtonMenuItem;
 import javax.swing.JScrollPane;
 import javax.swing.JTextArea;
+
 import com.game.src.main.classes.SaveFile;
 
 public class GameWindow extends Canvas implements Runnable { // This interface is useful when utilizing multiple threads

```

Comment:
I left out a lot of smaller comments that aren't as big of a priority, but one more thing I want to call out:

You have a _lot_ of logic in this GameWindow class, including logic that handles a lot of unrelated things:

- Initializing game state
- Updating game state (score, position, etc)
- Rendering
- Knowing where to get resources (files, etc).

You'll find it much easier to manage your code if you think about how you could split some of these things into separate files/classes.